Remormat all the toml file again to fix npm upload

ogoffart opened the pull request:
Commit cd6f2e2cf22f239a98ecbd54efe4e923c7222da9 reformated the .toml files but the formatter split some line that should have been kept in one line as other scripts were parsing the toml

Give a sane column with to avoid over-splitting of lines.

The broken script was `publish_nom_package.yaml` that changes the default in api/node/Cargo.toml

ogoffart commented on .mise/tasks.toml:
```diff
@@ -25,7 +25,7 @@ run = "cargo fmt --all"
 ["fix:toml:format"]
 description = "Run taplo format"
-run = "taplo format"
+run = "taplo format --option column_width=120"
```
This line was manually edited

ogoffart commented on api/node/Cargo.toml:
```diff
@@ -21,13 +21,8 @@ crate-type = ["cdylib"]
-  "backend-qt",
-  "accessibility",
-]
+# This line is edited by the publish_npm_package.yaml workflow. Make sure it is only one line
```
This line was manually added

ogoffart reviewed (commented):
The new default is more inline with the previous practice.

hunger commented:
Let me see whether https://github.com/slint-ui/slint/pull/8043 works out:

That should fix the issue, too, and should work pretty independent of the format we chose to store the file on disk. It basically reformats with infinite columns, runs the scripts and formats back.

tronical commented:
This seems like a reasonable compromise to me. We still enforce formatting, but it's slightly simpler.

tronical reviewed (approved):
To me this change is an improvement:

- It's visually more pleasing
- It adds comments about the automated editing being done
- It also fixes the packaging


I think #8043 makes sense in parallel, as it makes the automated editing more robust, by itself.

hunger commented:
I merged #8043, so the packaging should be fixed, only leaving the more pleasing and the added comment.

As usual I have no strong opinion on the specifics of the configuration of a code formatter: I do not have to worry about formatting stuff myself and fixing spaces in code review.

I would love the settings to move into .taplo.toml though, so people using the taplo LSP (e.g. via the taplo VSCode extension) also have the expected settings:

```toml
[formatting]
column_width = 120
```

should do this trick, but I did not test this.

ogoffart commented:
Rebased:

Added a .taplo.toml

I did two commit:
 First commit change the max line width,
 Second commit change another setting not to split the feature array.

What do you think?
